test/e2e/storage/drivers/in_tree.go

```diff
@@ -1139,9 +1154,13 @@ func InitGcePdDriver() testsuites.TestDriver {
 	)
 	return &gcePdDriver{
 		driverInfo: testsuites.DriverInfo{
-			Name:                 "gcepd",
-			InTreePluginName:     "kubernetes.io/gce-pd",
-			MaxFileSize:          testpatterns.FileSizeMedium,
+			Name:             "gcepd",
+			InTreePluginName: "kubernetes.io/gce-pd",
+			MaxFileSize:      testpatterns.FileSizeMedium,
+			SupportedSizeRange: volume.SizeRange{
+				Min: "5Gi",
+				Max: "64Ti",
```

Comment:
Previously the size was 5Gi, so it seems fine to put 5Gi as Min like this.
However how to make sure that 64Ti as Max?
That means I'd like to know how to define this Max value.
I could not find the reason of this Max value from the PR message and the corresponding issue report unfortunately.

Reply:
I would leave Max empty to indicate that we don't have a known value.

Reply:
I agree with @pohly here if we don't have a concrete reason at this time.

Reply:
thansk @pohly  @oomichi , I will fix it later

Reply:
By default, all max size should have not set now.